ddtrace/internal/remoteconfig/utils.py

```diff
@@ -1,5 +1,3 @@
-import os
```

Comment:
Same here. This seems like a merge conflict that should be resolved in the 2.x branch.

Reply:
This was also a fix for a poorly done rebase 😓